### PR TITLE
Add docstrings

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
It is looking much nicer with documentation included 😻

### CODE REVIEW

File: lib/simctl.js [hunk 1]
```diff
@@ -33,45 +54,145 @@ async function simCommand (command:string, timeout:number, args:Array = [], env
...
+ * @throws {Error} If the corresponding simctl subcommand command
+ *                 returns non-zero return code.
+ */
+async function spawn (udid, executablePath, env = {}) {
   return await simExec('spawn', 0, [udid, executablePath], env);
 }
 
-async function spawnSubProcess (udid:string, executablePath:string, env = {}):void {
+/**
+ * @alias spawn
```
Comment (imurchie) [inline lib/simctl.js]: This is not correct. `spawn` uses `exec` (which runs and returns a value) and `spawnSubProcess` creates a `SubProcess` (which does not run).

Comment (mykola-mokhnach) [reply lib/simctl.js]: Thanks, I'll fix that. Didn't know it myself %)

**Comment by imurchie** [comment]
Linting is still an issue.

**Comment by mykola-mokhnach** [comment]
@imurchie I don't observe any issues:

```
$ npm run lint

> node-simctl@3.11.0 lint /Users/elf/code/node-simctl
> gulp eslint

[14:06:19] Using gulpfile ~/code/node-simctl/gulpfile.js
[14:06:19] Starting 'eslint'...
[14:06:25] Finished 'eslint' after 6.69 s
```

**Comment by mykola-mokhnach** [comment]
Fixed trailing spaces  - should be better now

**Comment by imurchie** [comment]
Awesome!

**Pull request merged**